src/NetObject.hpp

```diff
@@ -24,6 +25,7 @@ namespace SokuLib
 		std::vector<struct ClientInfo> g_psvClients;
 	};
 
+    //! @brief The information exchanged during character select.
```

Comment:
I don't see what you mean by that. The NetObject is jus  the object responsible for handling packets and the network stuff in general.